maap/utils/JobUtils.py

```diff
@@ -0,0 +1,18 @@
+# Valid job statuses (loosely based on OGC job status types)
+JOB_STATUSES = ['Accepted', 'Running', 'Succeeded', 'Failed', 'Dismissed', 'Deduped', 'Offline']
```

Comment:
Since we have a fixed list of statuses that is used for look ups, you can use set() here.
```suggestion
JOB_STATUSES = {'Accepted', 'Running', 'Succeeded', 'Failed', 'Dismissed', 'Deduped', 'Offline'}
```